### PR TITLE
Implement navbar and basic html pages

### PR DESCRIPTION
**Pull request by joshflchan**
* Add basic html pages with updated css 
* Add navbar to each page _(is there a better way to do this in html?)_
* Uploaded resume and profile image

image

### CODE REVIEW

**Comment by joshflchan** [comment]
Is this a reasonable amount of changes for a PR? Or do you think I could've managed to add in the content for the blog and project html pages as well?

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -2,14 +2,22 @@
...
+      <a href="index.html">Josh Chan</a>
+      <a href="blog.html">Blog</a>
+      <a href="projects.html">Projects</a>
+      <a href="public/JoshChan_Resume.pdf">Resume</a>
+    </div>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
+      <h1>Josh's Portfolio</h1>
+      <img src="public/images/profile.jpg" alt="A very handsome photo of Josh Chan."/>
```
Comment (KevinWorkman) [inline portfolio/src/main/webapp/index.html:18]: I love that you provided an alt attribute, this is super important for accessibility!

File: portfolio/src/main/webapp/style.css [hunk 2]
```diff
@@ -7,3 +7,27 @@
 #greeting-container {
   margin-top: 20px;
 }
+
+.navbar a {
+  text-align: center;
```
Comment (KevinWorkman) [inline portfolio/src/main/webapp/style.css:12]: This is completely optional, but if you're curious, the Google CSS style guide technically recommends alphabetizing your CSS declarations: https://google.github.io/styleguide/htmlcssguide.html#Declaration_Order

I actually personally don't like this rule, so I ignore it for my own personal projects. Just thought you might find it interesting, but feel free to take it or leave it.

File: portfolio/src/main/webapp/style.css [hunk 3]
```diff
@@ -7,3 +7,27 @@
...
+}
+
+a:hover {
+  color: #bb86fc;
+}
+
+body {
+  background-color: #121212;
+  color: #ffffff;
+font-family: 'Roboto', sans-serif;
```
Comment (KevinWorkman) [inline portfolio/src/main/webapp/style.css:31]: One more minor note: this indentation is a little off

Comment (joshflchan) [reply portfolio/src/main/webapp/style.css:31]: Good catch, thank you!

**Pull request merged**